corehq/apps/sms/api.py

```diff
@@ -397,6 +399,9 @@ def process_incoming(msg, delay=True):
         msg.couch_recipient_doc_type = v.owner_doc_type
         msg.couch_recipient = v.owner_id
         msg.domain = v.domain
+        user = CommCareUser.get(v.owner_id)
```

Comment:
this will slow down every incoming sms. not sure there's a way around that though.